app/src/main/res/layout/list_tagged_organizer_item.xml

```diff
@@ -16,55 +16,47 @@
   ~ limitations under the License.
   -->
 
-
-<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:card_view="http://schemas.android.com/apk/res-auto"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  android:layout_marginLeft="8dip"
-  android:layout_marginRight="8dip"
-  android:layout_marginTop="8dip"
-  card_view:cardCornerRadius="5dp">
-
-  <LinearLayout
+<LinearLayout
     android:id="@+id/contentContainer"
+    xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Check style is looking for only java files for now. We can extend it to
include xml s as well.
And we will add editor config too I hope in the future.

On Mon, Jul 27, 2015, 4:16 PM Marcus Gabilheri <notifications@github.com>
wrote:

> In app/src/main/res/layout/list_tagged_organizer_item.xml
> <https://github.com/gdg-x/frisbee/pull/421#discussion_r35533578>:
>
> >      android:id="@+id/contentContainer"
> > +    xmlns:android="http://schemas.android.com/apk/res/android"
>
> Fixing that! Is there a way for the indentation to be added to the
> checkstyle to prevent this from happening again? (I mean for future
> contributors_
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/421/files#r35533578>.
>